src/plugins/kibana/server/routes/api/ingest/register_simulate.js

```diff
@@ -0,0 +1,25 @@
+const _ = require('lodash');
+const buildRequest = require('../../../lib/ingest_build_request');
+const processResponse = require('../../../lib/ingest_process_response');
+
+module.exports = function registerSimulate(server) {
+  server.route({
+    path: '/api/kibana/ingest/simulate',
+    method: 'POST',
+    handler: function (request, reply) {
+      const client = server.plugins.elasticsearch.client;
+      const pipeline = request.payload;
```

Comment:
Is this really the pipeline, or pipeline + docs? Calling it pipeline is misleading if it includes docs as well. `simulatePayload` or something like that might be better.